### ISSUE TITLE
App export only partially imported into Specify

### ISSUE THREAD

**Issue by RebekkaML** [NHMDenmark/Herbarium-Sheets-workstation#111]
The data sheet "NHMD_Herba_20231115_16_16_SS_corrected" seems to have been only partially imported into Specify, the first 105 entries (945699 - 945803) are missing from the database. They appear in the original DigiApp export and in the corrected version, but are not in the processed version. The corresponding Dummy records have been deleted from Specify though. It looks like the records were lost somewhere during the process.

I would like to import these before proceeding with #91 (recording subspecies, variety, forma for boxes 1-285).

**Comment by RebekkaML** [comment]
This is the datasheet:

[NHMD_Herba_20231115_16_16_SS_corrected.csv](https://github.com/user-attachments/files/16262049/NHMD_Herba_20231115_16_16_SS_corrected.csv)

**Comment by RebekkaML** [comment]
I reran the DigiApp export (_corrected) through OpenRefine and the GREL script and uploaded it into Specify. I deleted all records that were marked as "already in Specify" and found that also the last 10 records of the sheet were not in there yet, so there was a total of 115 records missing. 
I imported them into Specify and manually added the checktaxon flag to all new taxa since the export was from an old version of the DigiApp and this feature didn't work yet.  

The new processed datasheet was added to the import archive on the N-Drive as "NHMD_Herba_20231115_16_16_SS_processed_new".
